mtp_api/apps/notification/utils.py

```diff
@@ -2,25 +2,23 @@
 
 from django.utils import timezone
 
-from notification.constants import EMAIL_FREQUENCY
+from notification.constants import EmailFrequency
 
 
-def get_notification_period(email_frequency):
+def get_notification_period(email_frequency: EmailFrequency):
     """
     Returns start and end datetime of previous day, week, or month
     NB: only DAILY is currently supported in noms-ops and email-sending management command
     """
-    today = timezone.make_aware(
-        datetime.combine(timezone.localdate(), time.min)
-    )
-    if email_frequency == EMAIL_FREQUENCY.DAILY:
+    today = timezone.make_aware(datetime.combine(timezone.localdate(), time.min))
+    if email_frequency == EmailFrequency.daily:
         return today - timedelta(days=1), today
-    elif email_frequency == EMAIL_FREQUENCY.WEEKLY:
+    elif email_frequency == EmailFrequency.weekly:
         start_of_this_week = today - timedelta(days=today.weekday())
         return start_of_this_week - timedelta(days=7), start_of_this_week
-    elif email_frequency == EMAIL_FREQUENCY.MONTHLY:
+    elif email_frequency == EmailFrequency.monthly:
         first = today.replace(day=1)
         return (first - timedelta(days=1)).replace(day=1), first
-    elif email_frequency == EMAIL_FREQUENCY.NEVER:
+    elif email_frequency == EmailFrequency.never:
         return today, today - timedelta(days=1)
```

Comment:
how does this "never" frequency work?

just have end < start would make queries work? I guess it would be the same with `(today, today)` but maybe this is confusing as it sounds like the range is for "today".